DDDEastAnglia.Tests/Controllers/HomeControllerBuilder.cs

```diff
@@ -0,0 +1,28 @@
+using DDDEastAnglia.Controllers;
+using DDDEastAnglia.DataAccess;
+
+namespace DDDEastAnglia.Tests.Controllers
+{
+    public class HomeControllerBuilder
+    {
+        private IConferenceLoader conferenceLoader = new ConferenceLoaderBuilder().Build();
```

Comment:
I assume `ConferenceLoaderBuilder` creates a real implementation of `IConferenceLoader` and not a mocked one? I think for the default values here, I'd rather not use the Builder, so you can see from reading the class what you get by default.